client/signup/steps/domains/index.jsx

```diff
@@ -192,6 +193,10 @@ class DomainsStep extends React.Component {
 		} );
 	};
 
+	isDomainForAtomicSite = () => {
```

Comment:
This should be a normal class method instead of an arrow function property.

Reply:
oops. didn't catch your 'wish' before merging. I'll catch it in the next PR, likely rewording one. sorry bud. :-)